app/scripts/controllers/preferences.js

```diff
@@ -75,6 +75,7 @@ export default class PreferencesController {
       snapsAddSnapAccountModalDismissed: false,
       ///: END:ONLY_INCLUDE_IN
       isLineaMainnetReleased: false,
+      securityAlertsEnabled: false,
```

Comment:
This boolean is generic for security alerts, which makes me think that if any provider is enabled then this should be set to true. Is that the goal? Does not seem so when I look in metametrics.js (there we are setting the value to be sent to `['blockaid']` especifically).